modules/ROOT/pages/administration/databases.adoc

```diff
@@ -91,11 +91,24 @@ CREATE OR REPLACE COMPOSITE DATABASE name
 ALTER DATABASE name [IF EXISTS]
 {
 SET ACCESS {READ ONLY \| READ WRITE} \|
-SET TOPOLOGY n PRIMAR{Y\|IES} [m SECONDAR{Y\|IES}]
+SET TOPOLOGY n PRIMAR{Y\|IES} [m SECONDAR{Y\|IES}] \|
+SET OPTION option value
 }
 [WAIT [n [SEC[OND[S]]]]\|NOWAIT]
 ----
 
+[source, syntax, role="noheader"]
```

Comment:
FYI `role="noheader"` is not needed anymore. The default for a code block is to _not_ have a header, unless a title is provided with a `.` row preceding the code block.